examples/pytorch/rgcn/experimental/entity_classify_dist.py

```diff
@@ -229,6 +229,7 @@ def evaluate(g, model, embed_layer, labels, eval_loader, test_loader, all_val_ni
     global_results = dgl.distributed.DistTensor(labels.shape, th.long, 'results', persistent=True)
 
     with th.no_grad():
+        th.cuda.empty_cache()
```

Comment:
why do we need this?

Reply:
Clean some GPU memory cache. Otherwise the training is more likely to OOM

Reply:
@BarclayII Should this happen with the torch allocator?

Reply:
Theoretically it could happen but it should be quite rare because the only place where we allocate memory outside PyTorch is (temporary) workspace allocation.  Which PyTorch version are you using?

Reply:
BTW this is only called in evaluation.